app/services/core_data_connector/import/media_content.rb

```diff
@@ -109,6 +112,10 @@ def column_names
            name: 'import_url',
            type: 'VARCHAR',
            copy: true
+         }, {
+          name: 'content_warning',
```

Comment:
We'll just want to note in the release notes that this is a breaking change, as it will now require all `media_contents.csv` files to contain this column.